wwwroot/css/site.css

```
@@ -1,4 +1,4 @@
-html {
+﻿html {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)